codecov.yml

```diff
@@ -1,5 +1,6 @@
 ignore:
   - config/*
+  - ironfish-cli/*
```

Comment:
don't you need to remove it below too? Or does this work enough?

Reply:
Yea lets try that first, not sure if there will be a caching error